ci/tests/general/general_disk.py

```diff
@@ -196,6 +199,32 @@ def partition_disk(disk):
         return disk.partitions[0]
 
     @staticmethod
+    def unpartition_disk(disk, wait=True):
+        """
+        Return disk to RAW state
+        :param disk: Disk DAL object
+        :return: None
+        """
+        if len(disk.partitions) == 0:
+            pass
```

Comment:
Don't you want to return here instead?
Cause 'pass' will continue executing the code